virttest/shared/cfg/base.cfg

```diff
@@ -1195,3 +1195,14 @@ uuid_dimm = ""
 #pr_helper_props_helper1 = {"sock_path": "/path/to/qemu-pr-helper.sock",
 #                           "pidfile": "/path/to/qemu-pr-helper.pid"}
 #
+
+# IGVM configuration
+# When enable_igvm is set to "yes", the framework will automatically use IGVM instead of OVMF
+# as the firmware. This is currently only effective on q35 machine type and requires
+# SNP (Secure Nested Paging) to be enabled.
+# TODO: Consider converting this into a firmware variant in guest-hw.cfg for better
+#       organization, especially if ARM CCA and other platforms enable coconut-svsm
+#       support in the future.
+# enable_igvm = no
+# igvm_path = /usr/share/coconut-svsm
+# igvm_filename = coconut-qemu.igvm
```

Comment:
hi @PaulYuuu , is this a vm level param? 

e.g. different vms can use different igvm settings:
vm1:
igvm_filename = vm1.igvm
vm2:
igvm_filename = vm2.igvm
vm3:
enable_igvm = no

Reply:
Considering `igvm` means "Independent Guest Virtual Machine", I am not sure we still need to `vm_` prefix, as the name is point to VM and clear for me as it's the VM's firmware.